### PR TITLE
Add a few more script to enable better debugging, and update the README

### PR DESCRIPTION
**Pull request by warhod**
When helping @alexcusack troubleshoot today, it's clear that from time-to-time we need to turn off headless mode to really understand what's going on. This PR makes running the project in various debug mode easier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,8 +93,20 @@ We're using headless browser automation to emulate a form-based sign-on. This is
  6. Present accessible roles to the user (if more than one) and allow them to select the role to assume
  7. Use the STS API to [assume the role](http://docs.aws.amazon.com/cli/latest/reference/sts/assume-role-with-saml.html)
  8. Save the token information to the [AWS credentials file](https://blogs.aws.amazon.com/security/post/Tx3D6U6WSFGOK2H/A-New-and-Standardized-Way-to-Manage-Credentials-in-the-AWS-SDKs)
- 
-### Limitations
+
+## Found a bug?
+
+[CLICK ME ![bug](https://meetearnest.atlassian.net/secure/viewavatar?size=xsmall&avatarId=11000&avatarType=issuetype)](https://meetearnest.atlassian.net/secure/CreateIssue!default.jspa?pid=12600&issuetype=10004)
```
Comment (harsha-y) [inline README.md]: External users won't be able to use our Earnest Jira

**Comment by warhod** [comment]
Fixed URL

**Comment by warhod** [comment]
I approve!

**Comment by warhod** [comment]
Tested to work with latest master

**Pull request merged**